README.md

```diff
@@ -1 +1,166 @@
-# Hello_Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+-->
+
+<div align="center">
+  
+  <br/>
+
+  <h3><b>Mateoburvi's Hello Microverse!</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello_Microverse! <a name="about-project"></a>
+
+
+
+**[Hello_Microverse!]** is the first GitHub repository project created by me, Mateo Burneo in Microverse
+
+## 🛠 Built With <a name="built-with"></a>
+
+  - HTML
+  - CSS
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">CSS</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[present_a_text]**
+- **[add_color_with_CSS]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-Git <br />
+-Vscode
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+  cd my-folder <br />
+  git clone https://github.com/mateoburvi/hello-microverse.git.
+
+  
+
+### Install
+
+Install this project with:
+
+  cd my-project
+  gem install
+
+Note: this proyect does not require an instalation
+
+### Deployment
```

Comment:
Good job so far with the set-up of your README, however, you have removed two important sections of the readme which are the usages and run tests. Do consider adding them as they are both good for letting your project users know what the project is being used for and how to run tests in the project